### PR TITLE
What’s new page for Firefox 102 [fix #11797]

### LINKED ISSUE #11797: WNP for Firefox 102
## Description

Firefox 102 releases on 2022-06-28.

EN - Pocket
FR - Firefox newsletter
DE - Pocket Hits

FR/DE copy: https://docs.google.com/document/d/1yYaC9WndE_m8RgU0cH8lYnZSOnNRRNOBqc2egro1sDs/
FR/DE design: https://www.figma.com/file/8fVLbzzAd40lSSLuOwCfnl/EU_Firefox_WNP_21-22?node-id=1604%3A4

EN copy/design forthcoming.

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds whatsnew pages for English, French, and German for Firefox 102.0 (releasing 28 June).

EN - Pocket
FR - Firefox newsletter
DE - Pocket Hits

## Issue / Bugzilla link
#11797 

## Testing
http://localhost:8000/en-US/firefox/102.0/whatsnew/
http://localhost:8000/fr/firefox/102.0/whatsnew/
http://localhost:8000/de/firefox/102.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx102-de.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-sm">
+      <div class="wnp-main-image mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-pocket">Pocket</div>
+      <h1 class="wnp-main-title">Lass dich inspirieren!</h1>
+      <p class="wnp-main-tagline">Erhalte die besten Artikel aus Pocket direkt in dein Postfach – jeden Tag!</p>
+
+      <p class="wnp-main-cta">
+        <a class="mzp-c-button mzp-t-product" href="https://getpocket.com/explore/pocket-hits-signup?utm_medium=mozilla-websites&utm_source=getpocket.com&utm_campaign=pocket-hits&utm_content=whatsnew-de">Melde dich für unseren Newsletter an</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx102-de.html:36]: Is it worth adding some data attributes for GA here so we know how many people click the link?

**Comment by alexgibson** [comment]
A couple of contrast issues on the newsletter variation in dark mode:

Screenshot 2022-06-23 at 13-24-43 Les nouveautés de Firefox encore plus de respect de la vie privée et de protection

Screenshot 2022-06-23 at 13-25-07 Les nouveautés de Firefox encore plus de respect de la vie privée et de protection

**Review by alexgibson — CHANGES_REQUESTED** [review]
One small analytics suggestion and some colour contrast issues with dark mode, but otherwise everything looks good to me. r+wc

**Comment by craigcook** [comment]
Updated, and there was also a small copy tweak on the EN page so I'm glad this didn't merge already.

**Review by alexgibson — APPROVED** [review]
r+

**Pull request merged**